presets/4.3/tune/UAV_tech_5in.txt

```diff
@@ -0,0 +1,48 @@
+#$ TITLE: UAV Tech - 5in (~630g AUW)
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: OFFICIAL
+#$ KEYWORDS: 5in, 5", freestyle
+#$ AUTHOR: UAV Tech (Mark Spatz)
+#$ DESCRIPTION: Mark is a Betaflight contributor, youtube content creator, and professional tuner (www.theuavtech.com)
```

Comment:
could you please specify:
is that 4s, 6S or both? Can add 4S, 6S keywords too.
Is that for RPM filters ON or OFF? If this tune is for RPM-only, you can set dshot600/dshot300 and set bdir ON
In this case don't forget about
```
#$ INCLUDE_WARNING: misc/warnings/en/dshot.txt
#$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
```